index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benjamin Patin's Developer Portfolio</title>
+  </head>
+  <body>
+
+    <!--- Top navigation Bar --->
+    <div class="topnav">
+      <div class="left"></div>
+      <div class="nav-menu">
+        <a href="#news">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+      </div>
+      <div class= "right"></div>
+    </div>
+
+  <!--- Headline Section --->
+    <div class="headline">
+      <div class="headline-text1">Hey There, I am Benjamin Patin.</div>
+      <div class="headline-text2">I am a Full Stack developper.</div>
+      <div class="headline-details"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+don’t hestiate to contact me.</div>
+      <div class="social">
+
+        <!-- Add font awesome icons -->
+          <a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
+          <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
+          <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
+          <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
+          <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a>
```

Comment:
- [ ] **[OPTIONAL]** I can see that you implemented the list and did a great job while doing so! I would just like to mention that whenever you have a list of elements, it's a good practice to wrap it in a `<ul>` (unordered list). This way, you can follow the HTML semantics and optimize your code!